include/text.h

```diff
@@ -138,28 +138,37 @@ struct FontInfo
     u8 shadowColor:4;
 };
 
-struct GlyphWidthFunc{
+struct GlyphWidthFunc
+{
     u32 font_id;
     u32 (*func)(u16 glyphId, bool32 isJapanese);
 };
 
-struct KeypadIcon {
+struct KeypadIcon
+{
     u16 tile_offset;
     u8 width;
     u8 height;
 };
 
+struct __attribute__((packed)) TextColor
```

Comment:
`__attribute__((packed))` doesn't do anything in struct declares. you have to use it on the variable.

Reply:
No, it does. It will make all variables of that struct packed. Compile my code with and without it and you'll see it doesn't match without it.